filebeat/input/docker/input.go

```diff
@@ -68,8 +69,8 @@ func NewInput(
 		return nil, errors.New("Docker input requires at least one entry under 'containers.ids'")
 	}
 
-	for idx, containerID := range ids {
-		cfg.SetString("paths", idx, path.Join(config.Containers.Path, containerID, "*.log"))
+	for idx, containerID := range config.Containers.IDs {
+		cfg.SetString("paths", idx, path.Join(config.Containers.Path, strings.Join([]string{containerID, config.Containers.Suffix}, "")))
```

Comment:
With default suffix `*.log` (without the `/`) this could also be:
```suggestion
		cfg.SetString("paths", idx, path.Join(config.Containers.Path, containerID, config.Containers.Suffix))
```

Reply:
Or this is intended to support any suffix?